doc/connectivity/region_extraction.rst

```diff
@@ -34,50 +34,50 @@ which is already preprocessed and publicly available at
 datasets.
 
 
-.. literalinclude:: ../../examples/connectivity/plot_extract_regions_canica_maps.py
+.. literalinclude:: ../../examples/connectivity/plot_extract_regions_dictlearn_maps.py
     :start-after: # utilities
     :end-before: ################################################################################
 
 .. currentmodule:: nilearn.decomposition
 
-Data decomposition using Canonical ICA
-======================================
+Brain maps using Dictionary Learning
+====================================
 
-Here, we use :class:`CanICA`, a multi subject model to decompose previously
-fetched multi subjects datasets. We do this by setting the parameters in the
-object and calling fit on the functional filenames without necessarily
-converting each filename to Nifti1Image object.
+Here, we use :class:`DictLearning`, a multi subject model to decompose previously
+fetched multi subjects datasets into functionally defined maps. We do this by setting
```

Comment:
The neuroscientist might protest: why "functional"? There is no notion on inference that is even remotely related to function in a neuriobiological or psychological sense.
Alternative "spatiotemporally consistent whole-brain pattern" or similar